modules/central-services-public-config.adoc

```diff
@@ -271,52 +271,166 @@ When you disable autoscaling, the `minReplicas` and `maxReplicas` parameters do
 
 | `scanner.resources.limits.cpu`
-| The CPU limit for Scanner to override the default value.
+| The CPU limit for the StackRox Scanner to override the default value.
 
 | `scanner.dbResources.requests.memory`
-| The memory request for Scanner database deployment to override the default values.
+| The memory request for the StackRox Scanner database deployment to override the default values.
 
 | `scanner.dbResources.requests.cpu`
-| The CPU request for Scanner database deployment to override the default values.
+| The CPU request for the StackRox Scanner database deployment to override the default values.
 
 | `scanner.dbResources.limits.memory`
-| The memory limit for Scanner database deployment to override the default values.
+| The memory limit for the StackRox Scanner database deployment to override the default values.
 
 | `scanner.dbResources.limits.cpu`
-| The CPU limit for Scanner database deployment to override the default values.
+| The CPU limit for the StackRox Scanner database deployment to override the default values.
 
 | `scanner.image.registry`
-| A custom registry for the Scanner image.
+| A custom registry for the StackRox Scanner image.
 
-| `scanner.image.name`
-| The custom image name that overrides the default Scanner image name (`scanner`).
+| ` `
+| The custom image name that overrides the default StackRox Scanner image name (`scanner`).
 
 | `scanner.dbImage.registry`
-| A custom registry for the Scanner DB image.
+| A custom registry for the StackRox Scanner DB image.
 
 | `scanner.dbImage.name`
-| The custom image name that overrides the default Scanner DB image name (`scanner-db`).
+| The custom image name that overrides the default StackRox Scanner DB image name (`scanner-db`).
 
 | `scanner.dbNodeSelector`
-| Specify a node selector label as `label-key: label-value` to force Scanner DB to only schedule on nodes with the specified label.
+| Specify a node selector label as `label-key: label-value` to force the StackRox Scanner DB to only schedule on nodes with the specified label.
 
 | `scanner.dbTolerations`
-| If the node selector selects tainted nodes, use this parameter to specify a taint toleration key, value, and effect for Scanner DB. This parameter is mainly used for infrastructure nodes.
+| If the node selector selects tainted nodes, use this parameter to specify a taint toleration key, value, and effect for the StackRox Scanner DB. This parameter is mainly used for infrastructure nodes.
+
+|===
+
+[id="central-services-public-configuration-file-scannerv4_{context}"]
+== Scanner V4
+The following table lists the configurable parameters for Scanner V4.
+
+|===
+| Parameter | Description
+
+| `scannerV4.db.persistence.persistentVolumeClaim.claimName`
+|The name of the PVC to manage persistent data for Scanner V4.
+If no PVC with the given name exists, it will be created. The default value is `scanner-v4-db` if not set. To prevent data loss, the PVC is not removed automatically with Central`s deletion.
```

Comment:
I missed this table earlier :facepalm: 
```suggestion
If no PVC with the given name exists, it is created. The default value is `scanner-v4-db` if not set. To prevent data loss, the PVC is not removed automatically when Central is deleted.
```